app/depict/src/test/java/org/openscience/cdk/depict/DepictionTest.java

```diff
@@ -0,0 +1,49 @@
+package org.openscience.cdk.depict;
```

Comment:
Please add the missing copyright/license header here.

Reply:
Added as of commit abbd8a4.